en/techdocs/api/client/hoodie.account.md

```diff
@@ -3,27 +3,25 @@ layout: layout-api
 locales: en
 ---
 # hoodie.account
-**version:** 		*> 0.1.0* <br />
-**source:** 		*hoodie/src/hoodie/account.js*<br />
+**version:** 		*> 0.1.0*<br>
+**source:** 		*[hoodie.js/src/hoodie/account](https://github.com/hoodiehq/hoodie.js/tree/master/src/hoodie/account)*
 
-**<br />after reading this you will know:**
+The account object in the client-side Hoodie API covers all user and authentication-related tasks, and enables you to do previously complex operations, such as signing up a new user, with only a few lines of frontend code. Since [data in Hoodie is generally bound to a user](/en/hoodieverse/glossary.html#private-user-store), it makes sense to familiarise yourself with **account** before you move on to [store](/en/techdocs/api/client/hoodie.store.html).
```

Comment:
I wonder if we can replaces "authentication-related tasks" with something else, just because "tasks" have a meaning in Hoodie land